cli/cmd/tapRunner.go

```diff
@@ -494,6 +486,33 @@ func cleanUpRestrictedMode(ctx context.Context, kubernetesProvider *kubernetes.P
 		handleDeletionError(err, resourceDesc, &leftoverResources)
 	}
 
+	if config.Config.Tap.DaemonMode {
```

Comment:
Delete anyway, regardless of `config.Config.Tap.DaemonMode`.
Otherwise, there will issues if mizu is started in daemon mode, crashes and leaves resources, then run again without daemon mode. The daemon mode resources won't get cleaned.

Reply:
This will fail and then notify the users that mizu couldnt remove these resources, even when not relevant

Reply:
We should probably have `mizu clean` attempt to clean everything in this manner, will add to the follow up ticket